tests/classy_jest/fields/EventCost/EventCost.spec.tsx

```diff
@@ -1,35 +1,38 @@
 import * as React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import { userEvent } from '@testing-library/user-event';
 import { beforeEach, describe, expect, it, jest } from '@jest/globals';
 import { RegistryProvider } from '@wordpress/data';
-import { createRegistry } from '../../../../common/src/resources/packages/classy/store';
+import { getRegistry } from '@tec/common/classy/store';
 import EventCost from '../../../../src/resources/packages/classy/fields/EventCost/EventCost';
-import { Currency } from '../../../../common/src/resources/packages/classy/types/Currency';
-import { CurrencyPosition } from '../../../../common/src/resources/packages/classy/types/CurrencyPosition';
-
-type Selector = {
-	'core/editor': {
-		getEditedPostAttribute: ( attribute: string ) => any;
-	};
-	'tec/classy': {
-		getDefaultCurrency: () => Currency;
-	};
-};
+import { Currency } from '@tec/common/classy/types/Currency';
+import { CurrencyPosition } from '@tec/common/classy/types/CurrencyPosition';
 
 // Mock WordPress data store
 jest.mock( '@wordpress/data', () => {
+	// Mock state inside the mock function to avoid hoisting issues
```

Comment:
This is fine and it's not a blocker for this PR, but know the objective is to avoid mocking stores and modules in tests.

I understand you've updated what you found in place, but I want to  point this out for future reference.